docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -373,6 +340,18 @@ Deletes an entry by id and returns its value.
 
 Query parameters use the LHS bracket syntax (i.e. they are encoded using square brackets `[]`).
 
+The following parameters are available:
+
+| Operator           | Type          | Description                                           |
+| ------------------ | ------------- | ----------------------------------------------------- |
+| `sort`             | String/Array  | Sorting the response                                  |
```

Comment:
Great idea, thanks! 
I'd suggest you add links to sub-headings, as suggested in this example:
```suggestion
| [`sort`](#sorting)             | String/Array  | Sorting the response                                  |
```

Reply:
I did try this but you couldn't actually see it was a link :/ Let me know what you think since I set the link on the description instead